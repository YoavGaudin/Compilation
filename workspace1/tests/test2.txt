real integer write read while do if then else
main var call return defstruct extern

() [] {} , : ;

== <> < <= > >=
+ -
* /
=
&&
||
! \		\\single backspace is invalid

