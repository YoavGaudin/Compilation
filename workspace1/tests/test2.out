<real> <integer> <write> <read> <while> <do> <if> <then> <else>
<main> <var> <call> <return> <defstruct> <extern>

() [] {} , : ;

<relop,==> <relop,<>> <relop,<> <relop,<=> <relop,>> <relop,>=>
<addop,+> <addop,->
<mulop,*> <mulop,/>
<assign,=>
<and,&&>
<or,||>
<not,!> 
Lexical error: '\' in line number 12
